tests/trigger/test_arguments.py

```diff
@@ -0,0 +1,77 @@
+from pytest_mock_resources import create_postgres_fixture
+from sqlalchemy import Column, types
+
+from sqlalchemy_declarative_extensions import (
+    Function,
+    Functions,
+    Triggers,
+    declarative_database,
+    register_sqlalchemy_events,
+)
+from sqlalchemy_declarative_extensions.dialects.postgresql import Trigger
+from sqlalchemy_declarative_extensions.sqlalchemy import declarative_base
+from sqlalchemy_declarative_extensions.trigger.compare import compare_triggers
+
+_Base = declarative_base()
+
+
+@declarative_database
+class Base(_Base):  # type: ignore
+    __abstract__ = True
+
+    functions = Functions().are(
+        Function(
+            "count_arguments",
+            """
+            DECLARE
+                arg_name TEXT;
+            BEGIN
+                FOREACH arg_name in ARRAY TG_ARGV LOOP
+                    RAISE NOTICE 'Processing %', arg_name;
```

Comment:
The notice is pointless but I found it helpful during manual testing on the DB. I tried to consume the emitted message programmatically, but I didn't find a way to do so. Therefore I added the `arg_count` column.